### ISSUE TITLE
Transport pipes not registering compact machines

### ISSUE THREAD

**Issue by xenoxsis** [aidancbrady/Mekanism#5206]
#### Issue description:
Basic Logistical Transporter not recognizing connection inside compact machines. Not sure if this is a Mekanism issue or a Compact Machines issue, I do suspect the former.

#### Steps to reproduce:
1. Put something inside a compact machine (CM) that outputs items. (I use an actually additions ranged collector, to collect cobblestone)
2. Create a line of Basic Logistical Transporters to output to a CM tunnel (mine outputs to bottom) - Pull from collector, push to tunnel.
3. On the output side of the CM (again mine is bottom) - I put a Basic Logistical Transport. Pull from the CM, Push to Storage Drawers. - No items are output - The "pull" side on the cm, doesn't change appearance when set to pull.

If inside CM I use another form of pipes. The transporter on the outside works as intended. Or if I use transporters on the inside of CM, it works with a hopper on the outside. But Transport inside to transport outside does not work.

#### Version (make sure you are on the latest version before reporting):
**Forge:** 14.23.5.2768
**Mekanism:** 9.4.13-349
**Other relevant version:** 
Compact Machines: 3.0.12-b215
Storage Drawers: 1.12.2-5.3.7

**Comment by xenoxsis** [comment]
Update: Just noticed that if I put a logistical transport (item cable) on the inside, and a logistical transport (item cable) on the outside. It will register as universal cable (power/rf cable) instead. Not sure if this is an issue with CM or Mekanism.

**Comment by thiakil** [comment]
Please add screenshots of the intended setup.

**Comment by xenoxsis** [comment]
Sure I just tried to replace the machines with chests so there would be no issue with the machine "accepting" the item.
![image](https://user-images.githubusercontent.com/4257863/49245589-1b0a0e80-f413-11e8-9e31-5c086a668b2b.png)
![image](https://user-images.githubusercontent.com/4257863/49245631-3a08a080-f413-11e8-9bb9-3a2b3dbdd915.png)

First one is the input. The storage drawer contains wheat. I set the BLT to pull from the storage drawer and "push" to the CM. Though the cable didn't change shape when I set it to pull. I tried with a chest instead of the CM, that works fine (even if the input to the chest is not set to push).

On the second picture, the tunnel input is set to "pull" (it also doesn't change form) and the chest input is set to push. As you see on the picture, the tunnel wall registers not as a Basic Logistical Transporter but as Universal Cable, and seems to think it should expect RF, which is probably why the item transport doesn't work.
It might be the CM tunnel that's the problem of course.

Let me know if I can supply any more information.
